configs/katello/4.1.yaml

```diff
@@ -0,0 +1,113 @@
+---
+:project: katello
+:github_org: katello
+:releases:
+  :4.1.0:
+    :redmine_version_id: 1351
+:prior_releases:
+  :4.0.0:
+    :redmine_version_id: 1177
+  :3.17.3:
+    :redmine_version_id: 1365
+  :3.18.2:
+    :redmine_version_id: 1353
+  :3.18.2.1:
+    :redmine_version_id: 1384
+  :3.18.3:
+    :redmine_version_id: 1370
+:code_name:
+:mash_scripts:
+  - katello-4.1-rhel7
+  - katello-4.1-thirdparty-candlepin-rhel7
+  - katello-4.1-el8
+  - katello-candlepin-4.1-el8
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-4.1
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/2.5
+    :repo: https://github.com/theforeman/foreman-packaging.git
+  :hammer-cli-katello:
+    :branch: 1.0.0
+    :repo: https://github.com/Katello/hammer-cli-katello.git
+  :katello-host-tools:
+    :branch: 3.5.4
+    :repo: https://github.com/Katello/katello-host-tools.git
+  :katello-certs-tools:
+    :branch: 2.7.3
+    :repo: https://github.com/Katello/katello-certs-tools.git
+  :foreman_virt_who_configure:
+    :branch: 0.5.5
+    :repo: https://github.com/theforeman/foreman_virt_who_configure.git
+:ignores:
+  - 31308 # already in
+  - 31712 # not merged into 2.5-stable
+  - 31967 # for katello 3.18
+  - 31636 # migration fixes for katello 3.18
+  - 31902 # for katello 3.17, 3.18
+  - 31974 # for katello 3.18
+  - 32060 # for katello 3.18
+  - 32062 # for katello 3.18
+  - 32066 # for migration/3.18
+:strict_keys: true
+:gpg_key: AC2AD9F1
+:tags:
+  - name: katello-4.1-thirdparty-candlepin-rhel7
+    based_off: katello-thirdparty-candlepin-rhel7
+    build_target: katello-4.1-rhel7-build
+    arches:
+      - x86_64
+  - name: katello-candlepin-4.1-el8
+    based_off: katello-candlepin-nightly-el8
+    build_target: katello-4.1-el8-build
+    arches:
+      - x86_64
+  - name: katello-4.1-rhel7
+    based_off: katello-nightly-rhel7
+    helper_tags:
+      katello-4.1-thirdparty-rhel7: katello-thirdparty-rhel7
+      katello-4.1-rhel7-override: null
+    build_target: katello-4.1-rhel7-build
+    build_package_group_source_tag: katello-nightly-rhel7-build
+    arches:
+      - x86_64
+    inherits:
```

Comment:
What is the `inherits` section? should it have the same keys as the current nightly config? What do the numbers mean and what should their values be?

Reply:
per justin, I'm changing the versions to 2.5/4.1 and modeling it off of the `4.0` config. :+1: